vagrant-provision.sh

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env sh
 
-add-apt-repository ppa:chris-lea/node.js
+add-apt-repository -y ppa:chris-lea/node.js
 apt-get update
 apt-get install -y nodejs imagemagick python-sphinx python-yaml \
   python-tz python-virtualenv python-shelltoolbox python-tornado \
   python-gflags g++ xvfb git firefox
 
-npm install -g jshint@2.1.3 mocha-phantomjs@3.3.1 phantomjs@1.9.2-6 #mocha-phantomjs@3.2.0 phantomjs@1.9.1-0
-# mocha-phantomjs@3.3.1 phantomjs@1.9.2-6
\ No newline at end of file
+npm install -g jshint@2.1.3 mocha-phantomjs@3.3.1 phantomjs@1.9.2-6
```

Comment:
I am running newer versions in my non qa-for-this-lxc so maybe we should update these at some point